test-d/snapshot.ts

```diff
@@ -0,0 +1,18 @@
+import {expectError} from 'tsd';
+import test from '..';
+
+test('snapshot', t => {
+	t.snapshot({foo: 'bar'});
+	t.snapshot(null, 'a snapshot with a message');
+	expectError(t.snapshot('hello world', null));
+	// See https://github.com/avajs/ava/issues/2669
+	expectError(t.snapshot({foo: 'bar'}, {id: 'an id'}));
```

Comment:
I appreciate adding these tests for the type definition. I don't think we need to test that a removed feature has been removed, however.